classes/class3/homework.md

```diff
@@ -0,0 +1,41 @@
+# Homework 3
+In this homework you will build your first Express application, create some dynamic Handlebars templates, and store and load information in a MongoDB database using Mongoose.
+## Assignment
+Create an Express application that has the following routes:
+* GET `/cats/new` => Creates a new cat record. A cat should have a random age, a name, and a list of colors.
+  * These should be generated upon creation, not hardcoded.
+  * Optional: Display a verification that a new cat was created, perhaps by stating the details of the new cat.
+* GET `/cats` = > Shows a sorted list of cats by age. This should display their names, colors and age.
+  * The display doesn't have to be pretty as long as its clear. Feel free to explore some HTML formatting, but we will cover it more next time.
+* GET `/cats/bycolor/:color` => Shows a sorted list of cats by age that have a specific color, where `:color` is a parameter, such as "orange", or "grey" that specifies the color.
+* GET `/cats/delete/old` => Deletes the record of the oldest cat (send it to a nice farm in the country). The cat should not longer appear on any lists
```

Comment:
Addressed by explaining the error of our ways and the adding the reason why we are doing this here.